pkg/server/commands/list_objects_test.go

```diff
@@ -22,3 +31,124 @@ func TestNewListObjectsQuery(t *testing.T) {
 		require.Error(t, err)
 	})
 }
+
+func TestCheckDispatchCount(t *testing.T) {
+	ds := memory.New()
+	ctx := storage.ContextWithRelationshipTupleReader(context.Background(), ds)
+
+	tests := []struct {
+		name                  string
+		model                 string
+		tuples                []*openfgav1.TupleKey
+		objectType            string
+		relation              string
+		user                  string
+		expectedDispatchCount uint32
+	}{
+		{
+			name: "Test direct relation",
```

Comment:
```suggestion
			name: "test_direct_relation",
```

For consistency and easier test identification, we underscore test names. Though I admit it's not obvious and there _should_ be some way to programmatically enforce.